client/components/status/status.test.js

```diff
@@ -456,6 +456,31 @@ describe("<Status /> interactions", () => {
     await tick();
     expect(spyFn.mock.calls.length).toBe(0);
   });
+  it("should submit loginFormRef after login", async () => {
```

Comment:
please rename this to something more explicit, eg: `should perform captive portal login (submit loginFormRef)`